openedx/features/course_experience/views/course_home.py

```diff
@@ -120,8 +121,26 @@ def render_to_fragment(self, request, course_id=None, **kwargs):
             'is_enrolled': enrollment is not None,
             'is_staff': has_access(request.user, 'staff', course_key),
         }
+
+        allow_anonymous = CourseWaffleFlag(
```

Comment:
There are now 5-6 places in `lms` and `openedx` where this flag is being defined. Can you create one constant in https://github.com/edx/edx-platform/blob/master/openedx/features/course_experience/__init__.py and use it everywhere? (Make sure to follow the naming convention of the existing flags).

Reply:
Yeah, this crossed my mind as well, but fell through the cracks.

I'l do that now.